app/components/trainees/apply_trainee_data/view.html.erb

```diff
@@ -0,0 +1,59 @@
+<%= render PageTitle::View.new(title: "check_details.show") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLink.new(
+    text: "Back to draft record",
+    href: review_draft_trainee_path(@trainee),
+    html_attributes: {
+      id: "back-to-draft-record",
+    },
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+
+    <% if @form.errors.any? %>
+      <%= render "trainees/check_details/error_summary" %>
+    <% end %>
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-8">
+          <span class="govuk-caption-l">
+            Draft record <%= "for #{trainee_name(@trainee)}" if trainee_name(@trainee).present? %>
+          </span>
+          Trainee Data
+        </h1>
+
+        <%= render Trainees::RouteIndicator::View.new(trainee: @trainee) %>
+      </div>
+    </div>
+
+    <h2 class="govuk-heading-m">
+      Personal details
+    </h2>
+
+    <%= render Trainees::Sections::View.new(trainee: @trainee, form: @form, section: :personal_details, show_incomplete: true) %>
+
+    <%= render Trainees::Sections::View.new(trainee: @trainee, form: @form, section: :contact_details, show_incomplete: true) %>
+
+    <%= render Trainees::Sections::View.new(trainee: @trainee, form: @form, section: :diversity, show_incomplete: true) %>
+
+    <h2 class="govuk-heading-m">
+      Education
+    </h2>
+
+    <%= render Trainees::Sections::View.new(trainee: @trainee, form: @form, section: :degrees, show_incomplete: true) %>
```

Comment:
As `delete` is an option 
![image](https://user-images.githubusercontent.com/470137/117947860-7d67d880-b308-11eb-840d-19113e952c96.png)

It now looks weird 
![image](https://user-images.githubusercontent.com/470137/117947960-98d2e380-b308-11eb-863a-cc20bf6b3593.png)